setup.py

```diff
@@ -822,6 +805,19 @@ def run(self):
        "data_files":  data_files,
        "zip_safe":  False,
 }
+if STRIPPED:
+    PACKAGEDATA = {
+    "cmdclass":    cmdclass,
+    "packages":    ['pygame',
+                    'pygame.threads',
+                    'pygame._sdl2'],
+    "package_dir": {'pygame': 'src_py',
+                    'pygame._sdl2': 'src_py/_sdl2',
+                    'pygame.threads': 'src_py/threads'},
+    "ext_modules": extensions,
+    "zip_safe":  False,
```

Comment:
An aside... I wonder if we can get zip_safe egg(zip file) based pygame working again. It makes load time quite a bit faster with slow IO like on many android devices. Might be something to research in the future.